### ISSUE TITLE
It worked for me

### ISSUE THREAD

**Issue by brianlagunas** [Astralius/PrismEventAggregationProblem#1]
Sorry, but it worked for me:

![image](https://user-images.githubusercontent.com/835562/98030160-dab92500-1dcd-11eb-8232-62f3af442c73.png)

**Comment by Astralius** [comment]
@brianlagunas What I noticed is that it only works if you launch the app directly (= release mode without debugging). Once I ran it with VS 2019 Debug mode or Release mode with debugging, no events are received and "Received!" is not shown. We were scratching our heads about this in my development team, and finally, we figured out that it is the .NET GC that is the culprit here: event subscribers register via weak reference and are gone before being called: see this [https://stackoverflow.com/a/1811096](https://stackoverflow.com/a/1811096). I think that was the case in debug mode, too, as GC operates a bit differently between those two configurations (Debug/Release).

Anyhow, the problem is solved now, I needed to use this overloaded syntax:
`eventAggregator.GetEvent<SelectTypeEvent>().Subscribe(this.OnTypeSelected, keepSubscriberReferenceAlive: true);`
